airflow/jobs/scheduler_job_runner.py

```diff
@@ -807,8 +818,18 @@ def process_executor_events(
                 ti.pid,
             )
 
-            with Trace.start_span_from_taskinstance(ti=ti) as span:
-                cls._set_span_attrs__process_executor_events(span, state, ti)
+            active_ti_span = cls.active_spans.get(ti.key)
+            if active_ti_span is not None:
```

Comment:
Minor nitpick, feel free to ignore and resolve.  we can use the walrus operator to "assign in place" and simplify that a little now, if you want.

```suggestion

            if active_ti_span := cls.active_spans.get(ti.key) is not None:
```

Reply:
Done.

Reply:
It also needs parentheses, otherwise it's assigning the result of the boolean condition to the `active_ti_span`.

```
airflow/jobs/scheduler_job_runner.py:835: error: "Literal[True]" has no
attribute "end"  [attr-defined]
                    active_ti_span.end(end_time=datetime_to_nano(ti.end_da...
                    ^~~~~~~~~~~~~~~~~~
Found 1 error in 1 file (checked 2 source files)
```

Reply:
So it does.  Sorry about that.  Still getting used to the right way to use it, myself.